nominees/neonatal-nutrition-digital-accelerator-kit-project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "NEONATAL NUTRITION DIGITAL ACCELERATOR KIT PROJECT",
+  "aliases": [
+    "NNDAK"
+  ],
+  "description": "This project streamlines data collection for newborn nutrition to aid in decision-making. ",
+  "website": "https://play.google.com/store/apps/details?id=com.intellisoft.nndak&hl=en",
```

Comment:
Please provide a valid website link that talks about the digital solution unlike the current [Google Play page](https://play.google.com/store/apps/details?id=com.intellisoft.nndak&hl=en). Thanks.